rag/chat.py

```
@@ -0,0 +1,83 @@
+from langchain_community.document_loaders import PyPDFLoader, TextLoader
+from langchain_community.vectorstores import FAISS
+from langchain_openai import OpenAI
+from langchain_openai import OpenAIEmbeddings
+from icecream import ic
+
+# Configura tu clave de API para OpenAI
+os.environ["OPENAI_API_KEY"] = os.getenv("OPENAI_API_KEY")
+
+
+# Función para cargar documentos PDF y Markdown
+def cargar_documentos(pdf_paths=[], md_paths=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)